checkout.go

```diff
@@ -0,0 +1,103 @@
+package goshopify
+
+import (
+	"fmt"
+)
+
+const checkoutsBasePath = "admin/checkouts"
+
+// CheckoutService is an interface for interfacing with the checkout endpoints
+// of the Shopify API.
+// See: https://help.shopify.com/api/reference/checkout
+// See also: https://help.shopify.com/api/tutorials/sell-through-the-checkout-api#creating-a-checkout
+type CheckoutService interface {
+	Get(string, interface{}) (*Checkout, error)
+	Create(Checkout) (*Checkout, error)
+	// Update(Checkout) (*Checkout, error)
+}
+
+// CheckoutServiceOp handles communication with the checkout related methods of
+// the Shopify API.
+type CheckoutServiceOp struct {
+	client *Client
+}
+
+// Checkout represents a Shopify checkout
+type Checkout struct {
+	CreatedAt           string        `json:"created_at"`
+	Currency            string        `json:"currency"`
+	CustomerID          int           `json:"customer_id"`
+	Email               string        `json:"email"`
+	LocationID          int           `json:"location_id"`
+	OrderID             int           `json:"order_id"`
+	RequiresShipping    bool          `json:"requires_shipping"`
+	ReservationTime     int           `json:"reservation_time"`
+	SourceName          string        `json:"source_name"`
+	SourceIdentifier    interface{}   `json:"source_identifier"`
+	SourceURL           string        `json:"source_url"`
+	TaxesIncluded       bool          `json:"taxes_included"`
+	Token               string        `json:"token"`
+	UpdatedAt           string        `json:"updated_at"`
+	PaymentDue          string        `json:"payment_due"`
+	PaymentURL          string        `json:"payment_url"`
+	ReservationTimeLeft int           `json:"reservation_time_left"`
+	SubtotalPrice       string        `json:"subtotal_price"`
+	TotalPrice          string        `json:"total_price"`
+	TotalTax            string        `json:"total_tax"`
+	Attributes          []interface{} `json:"attributes"`
+	Note                string        `json:"note"`
+	Order               interface{}   `json:"order"`
+	PrivacyPolicyURL    string        `json:"privacy_policy_url"`
+	RefundPolicyURL     string        `json:"refund_policy_url"`
+	TermsOfServiceURL   string        `json:"terms_of_service_url"`
+	UserID              int           `json:"user_id"`
+	WebURL              string        `json:"web_url"`
+	TaxLines            []struct {
+		Price string  `json:"price"`
+		Rate  float64 `json:"rate"`
+		Title string  `json:"title"`
+	} `json:"tax_lines"`
```

Comment:
Could you re-use the [`TaxLine`](https://github.com/getconversio/go-shopify/blob/9e3d52acdc5314e805f6399b908ec21413d68af3/order.go#L192-L196) struct here?